proto/powergate/user/v1/user.proto

```diff
@@ -322,6 +322,11 @@ message AddrInfo {
   string address = 2;
   string type = 3;
   string balance = 4;
+  VerifiedClientInfo verified_client_info = 5;
+
+  message VerifiedClientInfo {
+    string remaining_datacap_bytes = 1;
```

Comment:
This is `string` since the native type is `BigInt`, so in some extreme case, the user could have a lot of verified deal capacity.